collect_app/src/test/java/org/odk/collect/android/widgets/AnnotateWidgetTest.java

```diff
@@ -165,13 +164,19 @@ public void whenPromptHasCurrentAnswer_showsInImageView() throws Exception {
 
     @Test
     public void markupButtonShouldBeDisabledIfImageAbsent() throws Exception {
```

Comment:
Method tweaked to match (essentially cosmetic?) splitting of answer strings. On general principle it should test against both ways a bad path could arrive?